notifications.bs

```diff
@@ -108,19 +108,42 @@ the <a>notification</a>, but then it should have less visual priority than the
 is not otherwise accessible to the end user, especially since notification platforms that do not
 support these features might ignore them.
 
-<p>A <a>notification</a> has an associated list of
-zero or more <dfn for=notification lt=action id=actions>actions</dfn>. Each
-<a for=notification>action</a> has an associated
-<dfn for=action id=action-title>title</dfn> and <dfn for=action id=action-name>name</dfn> and
-<em>can</em> have an associated <dfn for=action>icon URL</dfn> and
-<dfn for=action>icon resource</dfn>. Users may activate actions, as alternatives to
-activating the notification itself. The user agent must determine the
-<dfn>maximum number of actions</dfn> supported, within the constraints of the
-notification platform.
+<p>A <a>notification</a> has <dfn for=notification lt=action id=actions>actions</dfn>, 
+a <a for=/>list</a> of <a>notification actions</a>, initially empty.
+
+<p>A <dfn id=notification-action>notification action</dfn> defines a way the user can interact with
+a notification, as an alternative to activing the notification itself.
+
+<div dfn-for="notification action">
+  <p>A <a>notification action</a> has a <dfn id=notification-action-type>type</dfn>, that is
+  "<code>button</code>" or "<code>text</code>".
+  
+  <p class=note>Actions of type "<code>button</code>" can only be activated, whereas actions of type
+  "<code>text</code>" allow the user to input text during activation.
+
+  <p>A <a>notification action</a> has a <dfn id=notification-action-title>title</dfn> (a DOMString).
```

Comment:
Please don't change the ID. That'll affect folks linking to this.

Thanks for adding a type, but let's make that "a string".

Reply:
Done